CHANGELOG

```diff
@@ -1,6 +1,13 @@
 v3.6.5 (XXXX-XX-XX)
 -------------------
 
+* Fixed that, when performing a graph AQL query while a (graceful) failover for
+  the leader of the system collections is in progress, ArangoDB would report a 
+  "Graph not found error".
+  The real error, namely that an agency transaction failed, was swallowed in 
```

Comment:
@markuspf Was it really an agency transaction? I thought it was a "normal" transaction with a document lookup on the `_graphs` collection?